src/components/button-toggle-group/button-toggle-group.spec.js

```diff
@@ -56,5 +56,9 @@ describe('ButtonToggleGroup', () => {
         width: '48%'
       }, wrapper.find(ButtonToggleGroupStyle));
     });
+    it('renders ButtonToggle correctly with validation errors', () => {
+      const wrapper = render({ theme: smallTheme, errorMessage: 'error' }, TestRenderer.create).toJSON();
+      expect(wrapper).toMatchSnapshot();
```

Comment:
I think that could be tested just by looking for a component generated by the validation instead of creating a snapshot of the whole component.

Reply:
Initially I had this written as an `assertStyleMatch` which always failed, I think because the test couldn't a nested styled component. More detail here:

https://github.com/styled-components/styled-components/issues/1985

I tried a number of suggested fixes, none of which worked. Then I used a snapshot instead, and the test passed (you can see the error styling being added).

I'm at a bit of an impasse here, so if you have any suggestions, I'm happy to hear them.